config/config_test.go

```diff
@@ -171,6 +171,52 @@ var _ = Describe("Test full application config", func() {
 		})
 	})
 
+	Context("configure zmq", func() {
+		var (
+			zmqConfig *Config
+      producers map[string][]telemetry.Producer
```

Comment:
i think this is not formatted. Can you run `make format`?

Reply:
Done. I'll add a .git hook for this :)

Reply:
Also, I'll open up a PR after this but I think it would be super helpful if producers required a `Close` method. Seems like most of the underlying Conns support this, any thoughts?

Reply:
makes sense. go for it!